manifests/deploy/common-alm-values.yaml

```diff
@@ -1,20 +1,318 @@
 name: metering-operator.vlatest
+labels:
```

Comment:
No need to add `labels` to this file. It isn't a kubernetes manifest. Just a values file for helm.